packages/backend-core/src/redis/init.ts

```diff
@@ -6,17 +6,19 @@ let userClient: Client,
   appClient: Client,
   cacheClient: Client,
   writethroughClient: Client,
-  lockClient: Client
+  lockClient: Client,
+  socketClient: Client
 
 async function init() {
   userClient = await new Client(utils.Databases.USER_CACHE).init()
   sessionClient = await new Client(utils.Databases.SESSIONS).init()
   appClient = await new Client(utils.Databases.APP_METADATA).init()
   cacheClient = await new Client(utils.Databases.GENERIC_CACHE).init()
   lockClient = await new Client(utils.Databases.LOCKS).init()
-  writethroughClient = await new Client(
```

Comment:
Just for awareness - this is no longer in a database as the database selection was not working - so all writethrough data is currently in the default key space.

We could probably do with moving sessions out into a separate database to reduce scan load but its not an issue currently so no rush.